coordinator/Dockerfile

```diff
@@ -1,20 +1,30 @@
-FROM openjdk:22-ea-17-slim-bookworm
+FROM openjdk:21-jdk-slim AS builder
 
 RUN apt-get update \
-	&& apt-get install curl -y \
   && apt-get install unzip -y \
 	&& apt-get clean \
 	&& rm -rf /var/lib/apt/lists/*
 
+RUN mkdir -p src libs
+
+COPY --from=zip ./coordinator.zip /src
+
+RUN unzip /src/coordinator.zip -d libs/
+
+FROM openjdk:21-jdk-slim
+
 WORKDIR /opt/consensys/linea/coordinator
 
-# copy application
-COPY --from=zip ./coordinator.zip /opt/consensys/linea/coordinator/libs/
-RUN unzip libs/coordinator.zip -d libs/ && mv libs/coordinator/lib/** libs/ && rm -R libs/coordinator/
+RUN apt-get update \
+    && apt-get install curl -y \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
+    && mkdir -p /opt/consensys/linea/coordinator/libs/ \
+      /opt/consensys/linea/coordinator/logs \
+      /opt/consensys/linea/coordinator/tmp/prover/request \
+      /opt/consensys/linea/coordinator/tmp/prover/response
```

Comment:
I believe we could remove all the `/opt/consensys/linea/coordinator/` here as the WORKDIR is defined